src/styles/blocks/product-card.scss

```diff
@@ -0,0 +1,86 @@
+.product-card {
+  box-sizing: border-box;
+  height: 406px;
```

Comment:
According to the [Code Style Rules](https://mate-academy.github.io/layout_task-guideline/html-css-code-style-rules), fixed height on elements should generally be avoided to make the design more flexible and adaptable to content changes. Consider using min-height instead, or ensure that the design can handle variable content lengths without breaking.

Reply:
fix [this comment](https://github.com/mate-academy/layout_product-cards/pull/4792/files#r1780037050)
do not hardcode card height